src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DacKeyFieldDeclaration/KeyFieldDacTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.DacExtensionDefaultAttribute;
+using Acuminator.Analyzers.StaticAnalysis.DacKeyFieldDeclaration;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.DacKeyFieldDeclaration
+{
+	public class KeyFieldInDacTests : CodeFixVerifier
+	{
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new KeyFieldDeclarationAnalyzer();
+
+		protected override CodeFixProvider GetCSharpCodeFixProvider() => new KeyFieldDeclarationFix();
+
+		[Theory]
+		[EmbeddedFileData("CompoundKey.cs")]
+		public virtual void TestCompoundKey(string source) =>
```

Comment:
Use async version of the test methods, see our coding guidelines.
What is the purpose of marking all test methods as `virtual`?

Reply:
There are no tests for Expected files with suffix "ShouldNotShowDiagnostic"
@dropsonic  should they be added?